media/src/project-activity-components/panels/event-detail-panel.tsx

```diff
@@ -46,7 +56,9 @@ export const EventDetailPanel: React.FC<EventDetailPanelProps> = (
                     <span className={'lt-icons lt-button-back__icon'}>
                         <FontAwesomeIcon icon={faArrowLeft}/>
                     </span>
-                    <span className={'lt-button-back__text'}>Back</span>
+                    <span className={'lt-button-back__text'}>
+                        {activeLayerTitle && (<>Back to {activeLayerTitle}</>)}
```

Comment:
"Add layer name to event detail pane" - this works for the base layer, but not for the student response layers. Noted as TODO in code.